tests/conftest.py

```diff
@@ -15,16 +15,16 @@
 TEST_ROOT_DIR = os.path.abspath(os.path.dirname(__file__))
 TEST_DATA_DIR = os.path.join(TEST_ROOT_DIR, 'data')
 TEST_INGEST_OUTPUT_DIR = os.path.join(TEST_DATA_DIR,
-                                      'test_study',
+                                      'simple_study',
```

Comment:
Hmm so what are we actually using `test_study` for now? Do we need it still

Reply:
yes and yes